kdc/kerberos5.c

```diff
@@ -2046,10 +2048,179 @@ get_local_tgs(krb5_context context,
     return ret;
 }
 
-/*
- *
- */
+static int
+siva_any_to_certs(hx509_context context, const SignedData *sd, hx509_certs certs)
```

Comment:
We don't need the `siva_` prefix.  You'll get credit in the commit message, and if you'd like to provide a copyright notice with your name and the same license as in `LICENSE` then we'd add that to `LICENSE`.

Reply:
Haha I litter this all across my to make it really easy to grep and search inside vim for things I've modified. Not at all for the attribution or anything :)

I will clean up my PR once it's ready for submission.